semgrep-core/src/core/il/IL.ml

```diff
@@ -250,10 +250,13 @@ and instr_kind =
   | Assign of lval * exp
   | AssignAnon of lval * anonymous_entity
   | Call of lval option * exp (* less: enforce lval? *) * exp argument list
+  | CallAssign of lval * call_assign wrap * exp argument list
```

Comment:
I would prefer not to add another `CallXyz` here, we can just use `CallSpecial` for this and extend `call_special`.